packages/client/src/Loc.ts

```diff
@@ -2063,6 +2068,30 @@ export class ClosedLoc extends LocRequestState {
     }
 }
 
+export class ClosedIdentityLoc extends ClosedLoc {
+
+    get secrets(): Secret[] {
+        return [];
```

Comment:
Should be computed based on `this._secrets`?

Reply:
Of course. Thanks.